hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/Constants.java

```diff
@@ -415,6 +415,17 @@ private Constants() {
   public static final String S3_METADATA_STORE_IMPL =
       "fs.s3a.metadatastore.impl";
 
+  /**
+   * Whether to fail when there is an error writing to the metadata store.
+   */
+  public static final String FAIL_ON_METADATA_WRITE_ERROR =
+      "fs.s3a.metadatastore.fail.on.write.error";
+
+  /**
+   * Default value (false) for FAIL_ON_METADATA_WRITE_ERROR.
```

Comment:
should use `{@value}` instead of `false` in the javadoc like elsewhere in this file

Reply:
updated